src/test/java/com/softserveinc/ita/dkrutenko/SoftServeTests.java

```diff
@@ -2,69 +2,69 @@
-        softServeBasePage.clickAcceptCookieMessageButton();
-        softServeBasePage.clickHeaderMenuButton();
-        softServeBasePage.clickLanguageSwitcher();
+    @Test(dataProvider = "softServeChangeLanguageVerification")
+    public void verifyChangeLanguageFunction(String english, String german, String ukrainian) {
+        softServeMainPage.clickAcceptCookieMessageButton();
+        softServeMainPage.clickHeaderMenuButton();
+        softServeMainPage.clickLanguageSwitcher();
         
-        var getTitle = softServeBasePage.getTitle();
+        var getTitle = softServeMainPage.getTitle();
         var expectedTitle = getTitle.contains(english)
                 || getTitle.contains(german)
                 || getTitle.contains(ukrainian);
-        assertTrue(expectedTitle);
+        assertThat(expectedTitle);
 
-        softServeBasePage.clickHeaderMenuButton();
-        softServeBasePage.clickLanguageSwitcher();
-        assertTrue(expectedTitle);
+        softServeMainPage.clickHeaderMenuButton();
+        softServeMainPage.clickLanguageSwitcher();
+        assertThat(expectedTitle);
     }
 
     @DataProvider
-    public Object[][] softServeContactVerification() {
+    public Object[][] softServeContactUsFieldsVerification() {
         return new Object[][]{
-                {"Dmytro", "Krutenko", "dospecwork@gmail.com", "SoftServe Academy",
-                        "+380957125027", "test ''contact us'' page", "4"}};
+                { new UserRepository().getContactUsUser() }};
     }
 
-    @Test(dataProvider = "softServeContactVerification")
-    public void verifyContactUsPage(String firstName, String lastName, String email,
-                                    String company, String phone, String message, String expectedCategory) {
-        softServeBasePage.clickAcceptCookieMessageButton();
-        softServeBasePage.clickHeaderContactsMenuButton();
-        ContactUsPage contactUsPage = softServeBasePage.clickViewFullContactPage();
+    @Test(dataProvider = "softServeContactUsFieldsVerification")
+    public void verifyContactUsFields(User user) {
+        softServeMainPage.clickAcceptCookieMessageButton();
+        softServeMainPage.clickHeaderContactsMenuButton();
+        ContactUsPage contactUsPage = softServeMainPage.clickViewFullContactPage();
 
-        contactUsPage.fillContactPageFields(firstName, lastName, email, company, phone, message);
+        contactUsPage.fillContactPageFields(user);
         contactUsPage.clickFormInputModalMenu();
-        contactUsPage.selectFormInputModalMenu(expectedCategory);
+        contactUsPage.selectFormInputModalMenu(user.getExpectedCategory());
 
         contactUsPage.clickAcceptTermsAndPolicyCheckbox();
         contactUsPage.clickAcceptUpdatesAndOffersCheckbox();
         var actualEmail = contactUsPage.getAttributeFromEmailField();
-        assertEquals(actualEmail, email);
+        assertThat(actualEmail.equals(user.getEmail()));
```

Comment:
separate all asserts from code before and use such usage of asserts (make this changes everywhere)

assertThat(actualText)
.as("Message if test fail: Text should be: " + expectedText)
.equals(expectedText);

Reply:
done